### PR TITLE
First draft of the MultiSig Wallet

### LINKED ISSUE #3282: Proper multisig wallet support
- listing of balances
- first-class transfer of value
- ease of wallet deployment (proper parameter settings)

### PR DESCRIPTION
**Pull request by ngotchac**
Would hopefully close #3282 for now...

This is a first draft and a WIP for support of multi-signatures wallets (attaching existing ones and creating new ones)....

### CODE REVIEW

File: js/src/views/Wallet/wallet.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.confirmed {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 0.75em 0.5em 0;
+
+  > * {
+    // height: 32px;
```
Comment (jacogr) [inline js/src/views/Wallet/wallet.css]: ?

**Comment by gavofyork** [comment]
(for a separate PR) would be good to have a secondary wallet option of https://github.com/ethcore/contracts/blob/master/BasicWallet.sol

**Comment by ngotchac** [comment]
@gavofyork Yes, we talked about this with @jacogr ; it will be available

**Comment by jacogr** [comment]
@ngotchac As discussed, functionality-wise there are some gaps (e.g. existing contract attachment, simple parameters, editing of options) and there are some UX nigglies along the way (layout inconsistencies, informtaion better placed elsewhere). All these we chatted about.

However since we already trimmed this one down far as possible, it probably wouldn't make sense to get that in since it makes it un-reviewable again. So my suggestion is to get it in in the current form, which will also allow for some wider functionality review.

Will just triage with you again on my list of items and compare against yours so I don't log things unnecessarily.

**Comment by gavofyork** [comment]
@jacogr mark looks good, then?

**Comment by gavofyork** [comment]
functionality-wise, it's pretty sound.

**Pull request merged**